tests_async/unit/test__upload.py

```diff
@@ -614,108 +579,20 @@ def test__prepare_request_success_with_headers(self):
         assert new_headers is not headers
         expected_headers = {
             u"content-range": u"bytes 0-32/33",
-            u"content-type": BASIC_CONTENT,
+            u"content-type": sync_test.BASIC_CONTENT,
         }
         assert new_headers == expected_headers
         # Make sure the ``_headers`` are not incorporated.
         assert u"cannot" not in new_headers
 
-    @pytest.mark.parametrize("checksum", ["md5", "crc32c"])
-    def test__prepare_request_with_checksum(self, checksum):
```

Comment:
TODO: these were also added with the checksum changes. seems they should be here. possibly though only in sync? investigate.